extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -61,10 +61,11 @@ class FlowFileRepository : public core::Repository, public std::enable_shared_fr
       : FlowFileRepository(name) {
   }
 
-  FlowFileRepository(const std::string repo_name = "", std::string directory = FLOWFILE_REPOSITORY_DIRECTORY, int64_t maxPartitionMillis = MAX_FLOWFILE_REPOSITORY_ENTRY_LIFE_TIME,
+  FlowFileRepository(const std::string repo_name = "", std::string checkpoint_dir = FLOWFILE_CHECKPOINT_DIRECTORY, std::string directory = FLOWFILE_REPOSITORY_DIRECTORY, int64_t maxPartitionMillis = MAX_FLOWFILE_REPOSITORY_ENTRY_LIFE_TIME,
```

Comment:
Minor, but why not const ref strings?

Reply:
Done in [c0110dc](https://github.com/apache/nifi-minifi-cpp/pull/959/commits/c0110dca0937ac4af93e32d7ccab8347132183c7)